extensions/amp-story-education/0.1/amp-story-education.css

```diff
@@ -114,3 +137,147 @@ amp-story-education {
   font-weight: 600 !important;
   line-height: 44px !important;
 }
+
+@keyframes i-amphtml-tap-outer {
+  0% {
+    transform: scale(1);
+  }
+
+  24%, 26% {
+    transform: scale(0.8);
+  }
+
+  40% {
+    transform: scale(1.1);
+  }
+
+  55%, 100% {
+    transform: scale(1);
+  }
+}
+
+@keyframes i-amphtml-tap-inner {
+  0% {
+    opacity: 0;
+    transform: scale(1);
+  }
+
+  24%, 26% {
+    opacity: 0.5;
+    transform: scale(1);
+  }
+
+  40% {
+    opacity: 0.5;
+    transform: scale(2.2);
+  }
+
+  55%, 100% {
+    opacity: 0.0;
+    transform: scale(2.6);
+  }
+}
+
+@keyframes i-amphtml-swipe-outer {
+  0%, 4% {
+    width: 60px;
+    opacity: 0;
+    left: 80px;
+  }
+
+  40% {
+    width: 100%;
+    opacity: 1;
+    left: 0;
+  }
+
+  62% {
+    width: 60px;
+    left: 0;
+  }
+
+  80%, 100% {
+    width: 60px;
+    opacity: 0;
+    left: 0;
+  }
+}
+
+@keyframes i-amphtml-swipe-outer-rtl {
+  0%, 4% {
+    width: 60px;
```

Comment:
Did we try on a low end device to make sure that this is acceptable?

Reply:
I tried it on a Nexus 5 and I don't think one could tell the difference. For these tiny/slow animations on a very limited number of shapes when nothing else is going on on the screen, it's not a very big deal.